src/arch/arm/table_walker.hh

```diff
@@ -1196,7 +1196,7 @@ class WalkUnit : public ClockedObject
         BaseMMU::Mode mode;
         MMU::ArmTranslationType tranType;
 
-        TableWalker::Port *port;
+        [[maybe_unused]] TableWalker::Port *port;
```

Comment:
What's the error/warning you are getting here?

I am surprised you would mark this as being maybe unused considering it is definitely being used